base_rotation_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: base_rotation_joint
    joint_speed: 0.1
    max_torque: 0.9
    motor:
        id: 1
        init: 2048
        min: 1027
        max: 3078

shoulder_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: shoulder_joint
    joint_speed: 0.1
    max_torque: 0.9
    motor:
        id: 2
        init: 2048
        min: 2053
        max: 3074

elbow1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow1_joint
    joint_speed: 0.3
    max_torque: 0.9
    motor:
        id: 4
        init: 2048
        min: 805
        max: 3306

elbow2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow2_joint
    joint_speed: 0.3
    max_torque: 0.9
    motor:
        id: 5
        init: 2048
        min: 798
        max: 3364

wrist_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: wrist_joint
    joint_speed: 0.3
    max_torque: 0.9
    motor:
        id: 6
        init: 2048
        min: 30
        max: 4060

right_finger_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_finger_joint
    joint_speed: 0.5
    max_torque: 0.1
    motor:
        id: 7
        init: 2048
        min: 1484
        max: 3079

left_finger_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_finger_joint
    joint_speed: 0.5
    max_torque: 0.1
    motor:
        id: 8
        init: 2048
        min: 1035
        max: 2619
elevator_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: elevator_joint
    joint_speed: 0.0
    joint_torque_limit: 0.6
    motor:
        id: 9
        init: 2048
        min: 0
        max: 4095

